### ISSUE TITLE
Who is @DDD_Borat  ?

### ISSUE THREAD

**Issue by ylorph** [ToJans/AMA#4]

**Comment by ToJans** [comment]
### TL;DR

[@DDD_Borat](https://twitter.com/ddd_borat) is a fictitious alter ego created on twitter by a DDD'er that loves to expose misunderstandings in DDD concepts.

That was easy, but I'd assume your question probably should be:

### Who is behind @DDD_borat ?

I can try to (mis)guide your thinking process
- It is somebody who is meticulous on the content of his/her tweets (so that one rules me out ;) ).
- It is someone who knows DDD by heart, otherwise he/she wouldn't be able to make such canny, but insightful remarks.
  -I'd assume it's someone I should know (either IRL or online), as he/she is an early DDD'er (way earlier than me I'd assume), and the community was way smaller in the beginning, but I can't exactly pinpoint whom it might be. I have some assumptions, but your guess is as good as mine...

I do believe that a big part of the charm of the `???_borat` accounts is exactly the anonymity - like `the Stig` in `Top Gear`, so if I would know the identity I'd probably not tell you...

![This image might or might not contain any pointers to the true identity of @DDD_borat](http://cdndata.bigfooty.com/2014/10/86803_ca850de804bd122582cce2aee6dbeafc.jpg)

**Comment by ylorph** [comment]
marvelous answer
